packages/upload-media/README.md

```diff
@@ -0,0 +1,108 @@
+# `@wordpress/upload-media`
+
+Core media upload logic implemented with a custom `@wordpress/data` store.
+
+## API Reference
+
+### Actions
+
+The following set of dispatching action creators are available on the object returned by `wp.data.dispatch( 'core/upload-media' )`:
+
+<!-- START TOKEN(Autogenerated actions|src/store/actions.ts) -->
+
+#### addItems
+
+Adds a new item to the upload queue.
+
+_Parameters_
+
+-   _$0_ `AddItemsArgs`:
+-   _$0.files_ `AddItemsArgs[ 'files' ]`: Files
+-   _$0.onChange_ `[AddItemsArgs[ 'onChange' ]]`: Function called each time a file or a temporary representation of the file is available.
+-   _$0.onSuccess_ `[AddItemsArgs[ 'onSuccess' ]]`: Function called after the file is uploaded.
+-   _$0.onBatchSuccess_ `[AddItemsArgs[ 'onBatchSuccess' ]]`: Function called after a batch of files is uploaded.
+-   _$0.onError_ `[AddItemsArgs[ 'onError' ]]`: Function called when an error happens.
```

Comment:
Random question: What happens when an error happens for one file, do we stop the upload of the remaining ones?

Reply:
Only one of the files is stopped, the others continue. Same as today when you call `uploadMedia` from `media-utils`.